### ISSUE TITLE
Map - Keyboard/Focus Order accessibility 

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#516]
3. SC 2.1.1 A – Keyboard – “Number Dots” on map can be reached with the keyboard but not activated.
4. SC 2.1.1 A – Keyboard – “Click here to wake” on map cannot be reached with the keyboard
5. SC 2.4.3 A – Focus Order – “Map Markers Pop up – When the map marker is activated a popup opens
but the focus is not moved to the popup
6. SC 2.4.3 A – Focus Order – “Map Markers Pop up Close button – When the map marker popup is closed 
Library Accessibility Alliance // BTAA GeoPortal Platform
High-level accessibility review against WCAG2.1 AA
Deque Systems – [Date – 2023/3/30]
Page 5 of 10
focus does not return to the map marker
7. SC 2.4.6 AA –Descriptive Label – “Map Markers”– All the map marker buttons have the same accessible
name but represent different locations.
